includes/REST/AdminNoticeController.php

```diff
@@ -53,11 +62,13 @@ public function register_routes() {
 
     /**
      * Get dokan specific notices
+     * @param WP_REST_Request $request
      *
      * @return WP_REST_Response
      */
-    public function dokan_get_admin_notices() {
-        $notices = Helper::dokan_get_admin_notices();
+    public function dokan_get_admin_notices( WP_REST_Request $request ) {
+        $notice_scope = $request->get_param( 'scope' );
+        $notices        = Helper::dokan_get_admin_notices( $notice_scope );
```

Comment:
Retrieve all notices and perform filter operation here.

Reply:
vai, done.